### PR TITLE
Added custom Required error message on correction page

### PR DESCRIPTION
**Pull request by favour234**
- created customRequiredErrorMessage message descriptor.
- added customised error message for "Other" option in correction reason page.

### CODE REVIEW

File: packages/client/src/forms/validation.ts [hunk 1]
```diff
@@ -69,7 +69,12 @@ const getValidationErrors = {
     validators.push(...getFieldValidation(field as IDynamicFormField, values))
 
     if (field.required) {
-      validators.push(required(requiredErrorMessage))
+      // validators.push(required(requiredErrorMessage))
```
Comment (tahmidrahman-dsi) [inline packages/client/src/forms/validation.ts:72]: Can you please remove this comment?

**Review by tahmidrahman-dsi — CHANGES_REQUESTED** [review]
LGTM! A small change is requested, otherwise looks great

File: packages/client/src/forms/validation.ts (hunk 1 above)
Comment (favour234) [reply packages/client/src/forms/validation.ts:72]: okay

**Review by tahmidrahman-dsi — APPROVED** [review]

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]
@favour234 @tahmidrahman-dsi Although this change is valid, I'm quite reluctant to introduce a new field in the `FormFields` as we can already achieve this without it. I'm attaching a diff file which can enable us to achieve the same result. 
[changes.txt](https://github.com/opencrvs/opencrvs-core/files/9810043/changes.txt)

**Comment by favour234** [comment]
> @favour234 @tahmidrahman-dsi Although this change is valid, I'm quite reluctant to introduce a new field in the `FormFields` as we can already achieve this without it. I'm attaching a diff file which can enable us to achieve the same result. [changes.txt](https://github.com/opencrvs/opencrvs-core/files/9810043/changes.txt)

updated..

**Comment by Zangetsu101** [comment]
@favour234 The changes are looking good. It's just that you need to create a PR in farajaland with the new messages that you added.

**Comment by favour234** [comment]
> @favour234 The changes are looking good. It's just that you need to create a PR in farajaland with the new messages that you added or updated

alright..would do so

**Comment by favour234** [comment]
> > @favour234 The changes are looking good. It's just that you need to create a PR in farajaland with the new messages that you added or updated
> 
>  updated
https://github.com/opencrvs/opencrvs-farajaland/pull/375/files

File: packages/client/src/forms/correction/reason.ts [hunk 2]
```diff
@@ -71,13 +73,16 @@ export const correctRecordReasonSectionGroup: IFormSectionGroup = {
             name: 'reasonForChange',
             type: 'TEXT',
             label: {
-              defaultMessage: 'Reason for change',
+              defaultMessage: 'Reason for correction',
```
Comment (Zangetsu101) [inline packages/client/src/forms/correction/reason.ts:76]: This translation needs to be changed in your farajaland PR as well

File: packages/client/src/forms/correction/reason.ts [hunk 3]
```diff
@@ -71,13 +73,16 @@ export const correctRecordReasonSectionGroup: IFormSectionGroup = {
...
               id: 'form.field.label.reasonForChange',
               description: 'Label for input Reason for Change'
             },
             required: true,
             initialValue: '',
-            validate: [],
+
```
Comment (Zangetsu101) [inline packages/client/src/forms/correction/reason.ts:82]:
```suggestion
```

File: packages/client/src/i18n/messages/validations.ts [hunk 4]
```diff
@@ -211,6 +211,13 @@ const messagesToDefine: IValidationMessages = {
       'The error message that appears on required fields on new user form',
     id: 'validations.userform.required'
   },
+
```
Comment (Zangetsu101) [inline packages/client/src/i18n/messages/validations.ts:214]:
```suggestion
```

File: packages/client/src/forms/correction/reason.ts (hunk 2 above)
Comment (favour234) [reply packages/client/src/forms/correction/reason.ts]: updated

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**